frontend/src/routes/email/emails.ts

```diff
@@ -36,7 +34,7 @@ export const componentMap = {
     [EmailTemplate.CreateProjectRequest]: CreateProjectRequest,
     [EmailTemplate.ApproveProjectRequest]: ApproveProjectRequest,
     [EmailTemplate.UserAdded]: UserAdded,
-} satisfies Record<EmailTemplate, ComponentType>;
+} as const;
```

Comment:
one big benefit of `satisfies` is that there will be an error if it doesn't satisfy the type, now there will never be an error. Can we change this back? maybe with a relaxed type? Previously if a new template type was added this would error if you didn't have a mapping, but now it won't.

Reply:
@myieye - You're the one who changed this to `as const`; any thoughts? Do you know why that change was necessary? Or is it possible to get a `satisfies` clause here that will make Typescript happy?

Reply:
we'll make this `satisfies Record<EmailTemplate, Component<any>>` or something close, the key of the records must be `EmailTemplate` the component could be replaced with `any`